sources/packages/backend/apps/api/src/route-controllers/institution-locations/models/application.dto.ts

```diff
@@ -44,6 +47,15 @@ export class ActiveApplicationSummaryAPIOutDTO {
   applicationScholasticStandingStatus: ApplicationScholasticStandingStatus;
 }
 
+/**
+ * Query string options available for active applications summary.
+ */
+export class ActiveApplicationSummaryAPIQueryStringDTO extends ApplicationStatusPaginationOptionsAPIInDTO {
```

Comment:
just to double are we not using `APIInDTO` for query string STO?

Reply:
Up to us, I was trying to make some differences between the payloads from the body.
We can keep the `APIInDTO` if the team prefers.
@dheepak-aot @guru-aot @andrepestana-aot @sh16011993 thoughts?

Reply:
Renamed to `ActiveApplicationSummaryQueryStringAPIInDTO` to keep `APIinDTO` suffix.